src/browser_actions/popup.js

```diff
@@ -79,6 +87,12 @@ const onResetClick = () => {
     renderClippings();
 }
 
+const onDeleteClick = (id) => {
```

Comment:
can you refactor this to `onDeleteItemClick` so it will be more clear its deleting only one item